xla/stream_executor/gpu/gpu_device_info_test.cc

```diff
@@ -47,7 +45,7 @@ TEST(DeviceInfoTest, DeviceInfoMatches) {
   }
 
   TF_ASSERT_OK_AND_ASSIGN(Platform * platform,
-                          PlatformManager::PlatformWithName("CUDA"));
+                          PlatformManager::PlatformWithName(PLATFORM_NAME));
```

Comment:
You can use the `PlatformUtil` class to obtain the currently available GPU platform.

There is an example in this test: https://github.com/openxla/xla/blob/3c40cddc40fc43b197374825d954ff806c607a26/xla/stream_executor/gpu/gpu_kernel_test.cc#L42